docs/develop/ethereum-to-ton/solidity-vs-func.md

```diff
@@ -0,0 +1,79 @@
+
+Permanent storage data in TON Blockchain is stored as a cell. Cells play the role of memory in the stack-based TVM. A cell can be transformed into a slice, and then the data bits and references to other cells from the cell can be obtained by loading them from the slice. Data bits and references to other cells can be stored into a builder, and then the builder can be finalised into a new cell.
+
+## Data types
+
+Solidity includes the following basic data types:
+- Signed/Unsigned integers
+- Boolean
+- Addresses – used to store Ethereum wallet or smart contract addresses, typically around 20 bytes. An address type can be suffixed with the keyword “payable”, which restricts it to store only wallet addresses and use the transfer and send crypto functions.
+- Byte arrays – declared with the keyword “bytes”, is a fixed-size array used to store a predefined number of bytes up to 32, usually declared along with the keyword.
+- Literals – Immutable values such as addresses, rationals and integers, strings, unicode and hexadecimals, which can be stored in a variable.
+- Enums
+- Arrays (fixed/dynamic)
+- Structs
+- Mappings
+
+
+In case of FunC, the main data types are:
+- Integers
+- Cell – basic for TON opaque data structure, which contains up to 1,023 bits and up to 4 references to other cells
+- Slice and Builder – special objects to read from and write to cells,
+- Continuation – another flavour of cell that contains ready-to-execute TVM byte-code
+- Tuples – is an ordered collection of up to 255 components, having arbitrary value types, possibly distinct.
+- Tensors – is an ordered collection ready for mass assigning like: (int, int) a = (2, 4). A special case of tensor type is the unit type (). It represents that a function doesn’t return any value, or has no arguments.
+
+Currently, FunC has no support for defining custom types,
+
+## Declaring and using variables
+
+Solidity is a statically typed language, which means that the type of each variable must be specified when it is declared.
+
+```js
+uint test = 1; // Declaring an unsigned variable of integer type
+bool isActive = true; // Logical variable
+string name = "Alice"; // String variable
+```
+
+FunC is a more abstract and function-oriented language, it supports dynamic typing and functional programming style.
+
+```func
+(int x, int y) = (1, 2); // A tuple containing two integer variables
+var z = x + y; // Dynamic variable declaration 
+```
+
+FunC's type flexibility and function-orientation provide convenience and security when developing smart contracts. It offers flexible type management through automatic inference and controls data access through feature visibility.
```

Comment:
Didn't get where flexibility came from. It was mentioned that no custom types. Need to add more clear arguments